### DISCUSSION TITLE
select Values in Mysql

### DISCUSSION THREAD

**Discussion by RasulGafarov** [sqlalchemy/sqlalchemy#10309] [Usage Questions]
The following code:

```
my_select = select(
            Values(
                sqlalchemy.column("column_1", String),
                sqlalchemy.column("column_1", String),
                name="temp_1", literal_binds=True
            ).data([("a", "2"), ("b", "3")]))
```
generates sql:
```
SELECT temp_1.column_1, temp_1.column_1 AS column_1_1 
FROM (VALUES ('a', '2'), ('b', '3')) AS temp_1 (column_1, column_1)
```
this code causes an error because mysql (8.0.33) requires the following syntax:

SELECT temp_1.column_1, temp_1.column_1 AS column_1_1 
FROM (VALUES ROW('a', '2'), ROW('b', '3')) AS temp_1 (column_1, column_1)

https://dev.mysql.com/doc/refman/8.0/en/values.html

how to fix this error? Thank you in advance

**Comment by CaselIT** [comment]
Hi,

It seems that it's currently not supported.
The seems to be somewhat related https://github.com/sqlalchemy/sqlalchemy/issues/7228

**Reply by CaselIT** [reply]
I've created https://github.com/sqlalchemy/sqlalchemy/issues/10310
